storage/src/desktop/storage_reference_desktop.cc

```diff
@@ -230,7 +230,8 @@ Future<void> StorageReferenceInternal::DeleteLastResult() {
 // Handy utility function, since REST calls have similar setup and teardown.
 void StorageReferenceInternal::PrepareRequest(rest::Request* request,
                                               const char* url,
-                                              const char* method) {
+                                              const char* method,
+                                              const char* content_type) {
```

Comment:
Now that PrepareRequest supports content_type as a parameter, it would nice to update UpdateMetadata below to make use of this parameter so that content-type is set in a consistent way.

Reply:
Good idea.